### DISCUSSION TITLE
Question about recent strings.xml

### DISCUSSION THREAD

**Discussion by raschaa** [johnkeel-thork/DMD2#57] [Translations]
Could you give me some more info on the 3 lines with "measure_correction_factor".

I'm not quite sure I understand correctly what it is about and how to translate...

greets
ra

**Comment by johnkeel-thork** [comment]
It’s basically a calibration multiplier.
Roadbook totals and partials will be multiplied by that value.

Sometimes when you have a Roadbook that was made using a wheel sensor you notice that there is a difference from DMD2 GPS readings.

Imagine the PDF deviates -5% from DMD2 GPS readings. Then you can “calibrate” the reading by adding 1.05 in that field.

**Reply by raschaa** [reply]
Ah OK, thought it was something like that...
